File: webscraper.py
''' 
install requests 
    $ pip install requests
install BeautifulSoup 
    $ pip install beautifulsoup4
# install  lxml 
    $ pip install lxml

to run the script
    python webscraper.py
'''
import requests
from bs4 import BeautifulSoup

# Send a request to the webpage
url = 'https://www.example.com'
response = requests.get(url)

# Parse the HTML content
soup = BeautifulSoup(response.content, 'lxml')

# Extract information from the HTML content
title = soup.find('title').text
paragraphs = [p.text for p in soup.find_all('p')]

print(title)
print(paragraphs)

File: webscraper.js
/*
1.) initializa npm
 $ npm init 
- follow the prompts
2.)install cheerio and axios
    $ npm install axios cheerio
3.) Run the code in terminal
     $ node zip.js
*/
const axios = require('axios');
const cheerio = require('cheerio');

const url = 'https://example.com';

axios
  .get(url)
  .then((response) => {
    const html = response.data;
    const $ = cheerio.load(html);
    const title = $('title').text();
    console.log(title);
  })
  .catch((error) => {
    console.log(error);
  });
